app_dart/bin/server.dart

```diff
@@ -111,6 +113,12 @@ Future<void> main() async {
         authProvider,
         scheduler,
       ),
+      '/api/scheduler/requests': SchedulerRequestSubscription(
```

Comment:
request-subscription is still very generic, it can apply to any request handler dealing with pubsub. Can we use something the implies what the handler is actually doing? e.g build-request-subscription if you envision multiple subscription handlers for the scheduler.

Reply:
Updated to `/api/scheduler/batch-request-subscription` as this schedules `BatchRequest` to BuildBucket